planner/core/common_plans.go

```diff
@@ -468,6 +468,28 @@ func (e *Execute) rebuildRange(p Plan) error {
 			}
 		}
 	case *PointGetPlan:
+		if x.Path != nil {
+			if x.PartitionInfo != nil {
+				return errors.Errorf("Ban partition for now")
+			}
+			if x.Path.IsTablePath {
+				x.Path.Ranges, err = ranger.BuildTableRange(x.Path.AccessConds, sc, x.Path.PkCol.RetType)
+				if err != nil || len(x.Path.Ranges) != 1 || !x.Path.Ranges[0].IsPoint(x.ctx.GetSessionVars().StmtCtx) {
+					return errors.Errorf("Range is not point")
+				}
+				x.Handle = x.Path.Ranges[0].LowVal[0].GetInt64()
```

Comment:
How about replacing it with `x.Handle = kv.IntHandle(x.Path.Ranges[0].LowVal[0].GetInt64())`. It will cause compile error in some envs.